src/shared-components/alert/index.tsx

```diff
@@ -46,30 +59,35 @@ class Alert extends React.Component {
     duration: 3,
     truncateText: false,
     type: 'default',
-    onExit: () => undefined,
+    onExit: (): void => undefined,
   };
 
-  constructor(props) {
-    super(props);
-    this.contentText = React.createRef();
-  }
+  static Container = ({ children }: childrenProp): JSX.Element => (
+    <AlertsContainer>{children}</AlertsContainer>
+  );
+
+  contentText = React.createRef<HTMLDivElement>();
+
+  timer: number | undefined;
 
   state = {
     exiting: false,
     exited: false,
   };
 
-  componentDidMount() {
+  componentDidMount(): void {
     const { duration, ctaContent, truncateText } = this.props;
 
     // Truncate text logic
     if (truncateText) {
       const contentElement = this.contentText.current;
-      const wordsArray = contentElement.innerHTML.split(' ');
-      while (contentElement.scrollHeight > contentElement.offsetHeight) {
-        wordsArray.pop();
-        // eslint-disable-next-line prefer-template
-        contentElement.innerHTML = wordsArray.join(' ') + ' ...';
+      if (contentElement) {
+        const wordsArray = contentElement.innerHTML.split(' ');
+        while (contentElement.scrollHeight > contentElement.offsetHeight) {
+          wordsArray.pop();
+          // eslint-disable-next-line prefer-template
+          contentElement.innerHTML = wordsArray.join(' ') + ' ...';
```

Comment:
Should we just change this to using template strings while we're here? contentElement.innerHTML = \`${wordsArray.join(' ')} ...\`;